script.js

```diff
@@ -0,0 +1,201 @@
+  	alert("Tie!");
+  // return null;
+	}
+};
+
+
+function isXWinner(){
+	return winsRowX() || winsColumnX() || winsDiagonalX();
+}
+
+function isOWinner(){
+	return winsRowO() || winsColumnO() || winsDiagonalO();
+}
+
+
+var winsRowX = function(){
+	return allThreeX (board[0][0],board[0][1],board[0][2])||
+		allThreeX (board[1][0],board[1][1],board[1][2])||
+		allThreeX (board[2][0],board[2][1],board[2][2]);	
+};
+
+var winsColumnX = function(){
+	return allThreeX (board[0][0],board[1][0],board[2][0])||
+		allThreeX (board[0][1],board[1][1],board[2][1])||
+		allThreeX (board[0][2],board[1][2],board[2][2]);
+};
+
+var winsDiagonalX = function(){
+	return allThreeX(board[0][0],board[1][1], board[2][2])||
+		 allThreeX(board[2][0],board[1][1], board[0][2]);
+};
+
+var allThreeX = function(cellOne, cellTwo, cellThree) {
+
+  return (cellOne === 'x') && (cellTwo === 'x') && (cellThree === 'x');
+};
+
+var winsRowO = function(){
+	return allThreeO (board[0][0],board[0][1],board[0][2])||
+		allThreeO (board[1][0],board[1][1],board[1][2])||
+		allThreeO (board[2][0],board[2][1],board[2][2]);	
+};
+
+var winsColumnO = function(){
+	return allThreeO (board[0][0],board[1][0],board[2][0])||
+		allThreeO (board[0][1],board[1][1],board[2][1])||
+		allThreeO (board[0][2],board[1][2],board[2][2]);
+};
+
+var winsDiagonalO = function(){
+	return allThreeO (board[0][0],board[1][1], board[2][2])||
+		 allThreeO (board[2][0],board[1][1], board[0][2]);
+};
+
+var allThreeO = function(cellOne, cellTwo, cellThree) {
+  
+  return (cellOne === 'o') && (cellTwo === 'o') && (cellThree === 'o');
+
+};
+
+
+function currentPlayer(selectBox, position1, position2) {
+	selectBox.addEventListener('click', function(){
+		if ( count%2 === 0) {
+			console.log('Player Move');
+			count ++;
+			this.innerHTML = "x";
+			this.style.backgroundColor = "blue";
+			board[position1][position2] = "x";
+			console.log('count of X' + count);
+			getWinner();
+		
+			
+		} else{
+			console.log('Opponent Move');
+			count++;
+			this.innerHTML = "o";
+			this.style.backgroundColor = "red";
+			board[position1][position2] = "o";
+			console.log('count of O' + count);
+			getWinner();
+		}
+	});
+}
+
+currentPlayer(document.querySelector("#box-1"), 0, 0);
+currentPlayer(document.querySelector("#box-2"), 0, 1);
+currentPlayer(document.querySelector("#box-3"), 0, 2);
+currentPlayer(document.querySelector("#box-4"), 1, 0);
+currentPlayer(document.querySelector("#box-5"), 1, 1);
+currentPlayer(document.querySelector("#box-6"), 1, 2);
+currentPlayer(document.querySelector("#box-7"), 2, 0);
+currentPlayer(document.querySelector("#box-8"), 2, 1);
+currentPlayer(document.querySelector("#box-9"), 2, 2);
+
+
+
+	document.getElementById('clear').addEventListener('click', function(event){
+		event.preventDefault();
```

Comment:
event.preventDefault() is only necessary if the default event of the button is to refresh the page. This only happens with buttons of type 'submit', so it should be safe to remove this line.